landavailability/tests/webclient/test_views.py

```diff
@@ -165,7 +165,7 @@ def test_search_view_postcode(self):
             'nearest_broadband_fast': None
         }]
 
-        api_url = '{0}/api/locations/'.format(
+        api_url = '{0}/api/location-search'.format(
             settings.LAND_AVAILABILITY_API_URL)
```

Comment:
as I mentioned yesterday, could we change this back to /locations/ ? `location-search` is quite "anti-restful" approach and not consistent with all the other API endpoints we have